FlowerySnails/Yak-Social-Network
# Navbar and JSON FIXES

## Link to Ticket
Closes #1 

## Description of Proposed Changes
When user successfully logs in, the main landing page will display a header and NavBar
User WILL be able to click buttons to be redirected to appropriate areas on the site
## Steps to Test
Outline the steps to test

```
Install the following packages:
npm install react-router-dom
npm install bloomer
npm install bulma
```
```
Checkout my Branch
git fetch --all
git checkout RafNav
```

1. Run build
2. NavBar should have Site Name, Home, Profile, Notifications, Search, Logout fields/buttons
3. On hover, button should highlight and change colors
4. Basic Animations
5. Search Field can take text but has not functionality yet

**PLEASE NOTE:** We had to change the JSON file to include "Location" in the USER object AND took the quotes off of "true" in the POSTS object

## Impacted Areas in Application
List general components of the application that this PR will affect:

- nav.js
- nav.css
- search.js
- App.js
- App.css
- index.js
- yak.json

## Mentions @username
@jcarter0149 
@stjepad 
@Jonny-Riggs 

## Definition of Done
The project must be fully documented. This includes the following:
Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
Every class must be documented with purpose, author, and methods.
Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
The project must be able to run fully, and without errors, on each teammate's system.
Fulfills every requirement.
Every line of code has been peer reviewed.
For projects that require unit testing, core functionality must be identified and have at least one test for each.
